public/index.html

```diff
@@ -7,9 +7,18 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Reddit timer is a tool for content creators."
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
```

Comment:
You can even combine the two links into one. I guess you can separate the fonts with `|`. Not 100% sure though

Reply:
yes it works